### DISCUSSION TITLE
Custom source for charts

### DISCUSSION THREAD

**Discussion by ghost** [brian-mckeown/flightdash#41] [Ideas]
Hello!
I have a source of charts for all airports all around the world so to get the charts I go to a website and put an icao into the url. So for instance, it is something like https://cdn.example.com/charts/{ICAO}.pdf.
I would really love a feature so that users could enter their own chart sources in settings, and then access it from the efb.
I really appreciate your work.

**Comment by brian-mckeown** [comment]
H @nikolzaksalpaus ,
Thanks for the idea! Can you give me a specific example of a chart url or API service that you'd want to use? We can certainly look into expanding our charts service sources, but each source/service differs in the way they provide the charts, so we'd need to understand as much as possible about the service to build it into FlightDash.io. 
The current service does have limitations outside of the US charts tbough, so we'd definitely love to hear what other chart services people are using outside of the US!
Thanks again.

**Reply by brian-mckeown** [reply]
@nikolzaksalpaus - apologies for the delayed response. Yeah, Jeppesen charts would be great, but not free. Maybe some day in the future. Also, I did try finding more details on chartfox's API for FlightDash.io, but was unable to find it, and didn't hear back from their team when reaching out directly. Would be great to integrate with them directly. Currently, FlightDash.io charts are sourced from: https://www.aviationapi.com/
